oeps/oep-0032-arch-unique-identifier-for-users.rst

```diff
@@ -33,8 +33,12 @@ We will use the *LMS user_id* to uniquely identify a user in the Open edX system
 
 The *LMS user_id* should be used for all internal events. Additionally, the *LMS user_id* can be used for public events, REST APIs, and JavaScript APIs.
 
-.. _Django auth_user: https://docs.djangoproject.com/en/2.0/topics/auth/default/#user-objects
+.. warning::
+
+    A new external user id should still be used with third-parties under certain circumstances. This is especially important when the user id is sent along with other `Personally Identifiable Information (PII)`_. See the `ADR for the edx-platform external_user_ids Django app`_ for more details.
```

Comment:
[clarification] What does "when the user id is sent along with other PII" mean?  Doesn't this OEP recommend never sending PII with the user_id?

Reply:
Would it be more clear if it read "...a user id..." in place of "...the user id..."? The ADR states that there was a case where PII needed to be sent to a third-party, and sending the LMS user_id was not an option because then the LMS user_id would no longer be anonymous.

Again, I agree that this OEP should get a better update, but I didn't feel prepared for that, and I felt that this warning was better than what existed, because the only questions I've heard about the OEP were from people who happened to know about the ADR and were confused.

I'd be happy to help with a more formal update, but I think that would take some time and a decision to prioritize this.